languages/csharp/exercises/concept/exceptions/.docs/instructions.md

```diff
@@ -0,0 +1,29 @@
+In this exercise you will be building a simple integer calculator. To make matters simple, it is assumed the class can only handle positive numbers.
```

Comment:
The instructions are currently based on the older version of the code, correct? I'll let you work on updating this. Here are some things to keep in mind:

- The headings should describe what is to be implemented, not how (see [the docs](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docsintroductionmd)). So instead of having headers like "Implement method X", headings should look like "Handle overflow conditions in the calculator and provide enhanced information to the caller" Makes sense?
- I like how your instructions slowly build upon each other, which is the right way to go about this. I'd have separate tasks for each operation, and then separate tasks for handling the error cases. 
- Tasks must have an example bit of code that shows what to expect. It might be helpful to see how the [`user-defined-exceptions` exercise wrote its tasks](https://github.com/exercism/v3/blob/master/languages/csharp/exercises/concept/user-defined-exceptions/.docs/instructions.md).